README.md

```diff
@@ -56,6 +56,15 @@
 
 .NET for Apache Spark releases are available [here](https://github.com/dotnet/spark/releases) and NuGet packages are available [here](https://www.nuget.org/packages/Microsoft.Spark).
 
+### First-Party Extensions
+Some extended features are available via extension NuGet packages:
+* [Microsoft.Spark.Extensions.Azure.Synapse.Analytics](https://www.nuget.org/packages/Microsoft.Spark.Extensions.Azure.Synapse.Analytics/)
```

Comment:
You can remove this as we no longer support this in this repo.

Reply:
Removed.